### PR TITLE
chore(suite): update network symbol naming 3

### PR DESCRIPTION
**Pull request by adderpositive**
## Description
- update network symbol naming

## Related 
Follow up https://github.com/trezor/trezor-suite/pull/15611

Related to #14725

### CODE REVIEW

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty.tsx [hunk 1]
```diff
@@ -51,15 +51,15 @@ export const AccountEmpty = ({ account }: AccountEmptyProps) => {
...
+                        <Translation id="TR_RECEIVE_NETWORK" values={{ network: symbolTicker }} />
                     ),
                 },
                 {
                     'data-testid': '@accounts/empty-account/buy',
                     key: '2',
                     onClick: handleNavigateToBuyPage,
                     children: (
-                        <Translation id="TR_BUY_NETWORK" values={{ network: networkSymbol }} />
+                        <Translation id="TR_BUY_NETWORK" values={{ network: symbolTicker }} />
```
Comment (tomasklim) [inline packages/suite/src/views/wallet/transactions/components/AccountEmpty.tsx]: can we just do symbol: symbol.toUpperCase()?

**Review by tomasklim — CHANGES_REQUESTED** [review]

File: packages/suite/src/actions/wallet/__fixtures__/coinjoinAccountActions.ts [hunk 2]
```diff
@@ -30,11 +30,7 @@ export const createCoinjoinAccount = [
             account: { accountType: 'coinjoin' },
         },
         result: {
-            actions: [
-                COINJOIN.CLIENT_ENABLE,
-                COINJOIN.CLIENT_DISABLE,
-                notificationsActions.addToast.type,
-            ],
+            actions: [],
```
Comment (tomasklim) [inline packages/suite/src/actions/wallet/__fixtures__/coinjoinAccountActions.ts]: so the tests were broken? 😀

Comment (adderpositive) [reply packages/suite/src/actions/wallet/__fixtures__/coinjoinAccountActions.ts]: The tests were ok, but do we need to call any of these actions, if the symbol is not supported?

File: packages/suite/src/views/wallet/transactions/components/AccountEmpty.tsx (hunk 1 above)
Comment (adderpositive) [reply packages/suite/src/views/wallet/transactions/components/AccountEmpty.tsx]: ok

**Review by tomasklim — APPROVED** [review]

**Pull request merged**